rollup.config.js

```diff
@@ -7,6 +7,8 @@ import glob from 'fast-glob'
 import {visualizer} from 'rollup-plugin-visualizer'
 import postcss from 'rollup-plugin-postcss'
 import packageJson from './package.json'
+import postcssCustomPropertiesFallback from 'postcss-custom-properties-fallback'
+const importedJSONFromPrimitives = require('./node_modules/@primer/primitives/tokens-next-private/fallbacks/color-fallbacks.json')
```

Comment:
Can we import directly from the package or is that not a supported path?

```suggestion
const importedJSONFromPrimitives = require('@primer/primitives/tokens-next-private/fallbacks/color-fallbacks.json')
```

This only comes up as we are looking into how to structure this as a set of workspaces and a package may not be installed as a sibling to this build config.

Reply:
Probably yes! Let me try it 😄